.github/workflows/integration_tests.yaml

```diff
@@ -0,0 +1,41 @@
+name: XCUI Tests
+on:
+  workflow_dispatch:
+    inputs:
+      SDK_TARGET_VERSION:
+        description: "Target a specific SDK version. Otherwise latest"
+        required: false
+  push:
```

Comment:
Unless I'm reading this wrong, we aren't triggering anything here?

Reply:
Leaving it empty means that every commit (push) will trigger this CI. All branches included 💅